libs/blocks/accordion/accordion.js

```diff
@@ -0,0 +1,80 @@
+import createTag from '../../utils/utils.js';
+
+const faq = { '@context': 'https://schema.org', '@type': 'FAQPage', mainEntity: [] };
+
+function setSEO(questions) {
+  faq.mainEntity.push(questions.map(({ name, text }) => (
+    { '@type': 'Question', name, acceptedAnswer: { text, '@type': 'Answer' } })));
+  const script = createTag('script', { type: 'application/ld+json' }, JSON.stringify(faq));
+  document.head.append(script);
+}
+
+function handleClick(el, dt, dd) {
+  const expanded = el.getAttribute('aria-expanded') === 'true';
+  if (expanded) {
+    el.setAttribute('aria-expanded', 'false');
+    dd.setAttribute('hidden', true);
```

Comment:
nit: can just set to empty string as it only needs to be present
```suggestion
    dd.setAttribute('hidden', '');
```

One thing to note is that the `hidden` attribute has very low specificity so if any css targets this it may get displayed.